nf_core/pipeline-template/.github/CONTRIBUTING.md

```diff
@@ -31,8 +31,11 @@ If you're not used to this workflow with git, you can start with some [docs from
 
 ## Tests
 
-You can optionally test your changes by running the pipeline locally. Then it is recommended to use the `debug` profile to
-receive warnings about process selectors and other debug info. Example: `nextflow run . -profile debug,test,docker --outdir <OUTDIR>`.
+You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
+
+```
+nf-test test --profile debug,test,docker --verbose
```

Comment:
I think we should use the `nf-core --verbose modules/swf test` command here, since it also tests stability